src/components/select-field/BaseSelectField.js

```diff
@@ -381,34 +399,47 @@ class BaseSelectField extends React.Component<Props, State> {
             listboxProps['aria-multiselectable'] = true;
         }
 
+        const dropdownPlacement = isRightAligned ? 'bottom-end' : 'bottom-start';
```

Comment:
We'll eventually want these as importable constants, since they will be shared with all pop-ing things like drop downs and tooltips etc.

Reply:
I'll create this in  `/common/constants.js` for now, unless there's a better place you had in mind?

Reply:
Should we have a PopperComponent, similar to TetherComponent, that can encapsulate the options and their string values?